### PR TITLE
Add allPlayersOfPlayerType

### PR DESCRIPTION
**Pull request by bruh-9000**

### CODE REVIEW

File: src/gameClasses/components/script/VariableComponent.js [hunk 1]
```diff
@@ -2046,6 +2046,18 @@ var VariableComponent = TaroEntity.extend({
...
 
+					case 'allPlayersOfPlayerType':
+						var playerType = self.getValue(text.playerType, vars);
+
+						returnValue = (taro.$$('player'))
+							.filter((player) => {
+								return player._stats.playerTypeId === playerType;
+							});
+
+						console.log(returnValue.length);
```
Comment (sloont) [inline src/gameClasses/components/script/VariableComponent.js:2057]: remove log

File: src/gameClasses/components/script/VariableComponent.js [hunk 2]
```diff
@@ -2046,6 +2046,18 @@ var VariableComponent = TaroEntity.extend({
 
 						break;
 
+					case 'allPlayersOfPlayerType':
+						var playerType = self.getValue(text.playerType, vars);
+
+						returnValue = (taro.$$('player'))
```
Comment (sloont) [inline src/gameClasses/components/script/VariableComponent.js:2052]: `taro.$$('player')` doesn't need to be inside `(...)`

Comment (bruh-9000) [reply src/gameClasses/components/script/VariableComponent.js:2052]: What is (...)?

**Comment by bruh-9000** [comment]
@sloont

**Comment by bruh-9000** [comment]
https://github.com/moddio/moddio2/commit/a84e4340e30ce47f9b8ff25e1dc5b87a3c869c9d

Apparently m0dE added it lol...

**Comment by minotalen** [comment]
also close since @m0dE added
we need to review faster :/

**Pull request closed without merging**